WTWSMS/events/ACR_celtic_flavor_events.txt

```diff
@@ -348,7 +354,27 @@ character_event = {
 		completely_controls = c_innse_gall
 		
 		NOT = { religion_group = pagan_group }
-		culture_group = celtic
+		OR = {
+			culture_group = celtic
+			culture = romano-british
+			culture = saxon
+			culture = jute
+			culture = angle
+			culture = anglo-saxxon
+			culture = english
+			culture = norman
+			culture = old_saxon
+			culture = briton
+			culture = breton
+			has_landed_title = c_innse_gall
+		}
+
+		NOT = {
+			has_game_rule = {
+				name = supernatural_events
+				value = off
+			}
+		}
```

Comment:
I made that Event conditional to the Supernatural Rule. In it, a character who rules that island, is Celtic, or is in the vicinity of that island can go and be cured of any disease by Celtic Pagan white magic. You then have the option to convert to Celtic Paganism. That is a definitively supernatural event.